reset.sh

```diff
@@ -1,4 +1,19 @@
 #!/usr/bin/env bash
+set -euo pipefail
 
-./clean.sh "$1"
-./install.sh
+# Variables needing default values set in order to source install/error-handling
+MINIMIZE_DOWNTIME="${MINIMIZE_DOWNTIME:-}"
+STOP_TIMEOUT=60
+MSYSTEM="${MSYSTEM:-}"
+
+# Save logs in order to send envelope to Sentry
+log_file=sentry_reset_log-$(date +'%Y-%m-%d_%H-%M-%S').txt
+exec &> >(tee -a "$log_file")
+
+# Source files needed to set up error-handling
+source install/dc-detect-version.sh
+source install/detect-platform.sh
+source install/error-handling.sh
+trap_with_arg cleanup ERR INT TERM EXIT
+source docker_cleanup.sh
```

Comment:
`./clean.sh "$1"` -> `source docker_cleanup.sh`

sourcing docker_cleanup.sh will automatically use the version in $1 so functionality is still the same